blosc/shuffle-altivec.c

```diff
@@ -15,61 +16,145 @@
 #endif
 
 #include <emmintrin.h>
-
-
-/* The next is useful for debugging purposes */
-#if 0
+#include <altivec.h>
 #include <stdio.h>
-#include <string.h>
 
-static void printxmm(__m128i xmm0)
+
+//Store a vector to an unaligned location in memory
```

Comment:
Add a space between the '//' and the actual comment (similar to the space after the ' #' comment in Python).